app/views/widgets/textfields.html

```diff
@@ -9,7 +9,7 @@
         {{ fieldError(field) }}
 
         <label class="form-label" for="{{id}}">{{text | safe}}</label>
-        <input type="text" class="form-control {{ 'form-control-error' if field.error }} form-control-3-4" id="{{id}}" name="{{name}}" value="{{field.value  | safe}}" maxlength="{{size}}">
+        <input type="text" class="form-control {{ 'form-control-error' if field.error }} form-control-3-4" id="{{id}}" name="{{name}}" value="{{field.value  | safe}}" maxlength="{{size}}" size="{{size}}">
```

Comment:
`The size attribute specifies the visible width, in characters, of an <input> element. Note: The size attribute works with the following input types: text, search, tel, url, email, and password. Tip: To specify the maximum number of characters allowed in the <input> element, use the maxlength attribute.`

And we already have the `maxlength` attribute, so I'd say the `size` attribute is not necessary.